llvm/test/Transforms/PGOProfile/memprof_annotate_yaml.test

```diff
@@ -0,0 +1,45 @@
+; Make sure that we can ingest the MemProf profile in YAML and
+; annotate a call to new as cold.
+
+; REQUIRES: x86_64-linux
```

Comment:
Prefer putting the REQUIRES lines at the top before everything else.

Reply:
Fixed.